features/users/user_management.feature

```diff
@@ -58,4 +58,15 @@ Feature: Create and maintain projects
       | Documentation | My documentation | hello world |
     When I delete my profile
     And I am on the "Show" page for project "hello world"
-    Then I should see "by Anonymous"
\ No newline at end of file
+    Then I should see "by Anonymous"
+
+  @omniauth
+  Scenario: Unlink my github link from my profile
```

Comment:
make sense. Although there is controller level spec to test that. But its good to have a acceptance level test in place.